compiling dbi.c with '-Werror' fails (dbi_initialize, dbi_driver_open, 
dbi_driver_list are deprecated in recent libdbi)
--- collectd-4.9.1/src/Makefile.am.orig	2010-04-08 15:49:05.024241083 +0200
+++ collectd-4.9.1/src/Makefile.am	2010-04-08 15:52:22.184768228 +0200
@@ -265,6 +261,7 @@ if BUILD_PLUGIN_DBI
 pkglib_LTLIBRARIES += dbi.la
 dbi_la_SOURCES = dbi.c \
 		 utils_db_query.c utils_db_query.h
+dbi_la_CFLAGS = -Wall
 dbi_la_CPPFLAGS = $(AM_CPPFLAGS) $(BUILD_WITH_LIBDBI_CPPFLAGS)
 dbi_la_LDFLAGS = -module -avoid-version $(BUILD_WITH_LIBDBI_LDFLAGS)
 dbi_la_LIBADD = $(BUILD_WITH_LIBDBI_LIBS)
